### PR TITLE
Add door times

### LINKED ISSUE #255: Backend should provide start/stop times for the links in the schedule
To avoid possible confusion and skew, since the Backend is already computing precisely who is allowed into meeting when anyway, it should send the timing info to the Frontend in the Schedule.

Note that the key goal here is to say when the public is allowed to enter the meeting.  For Prep Sessions, the start time should be listed as the time of the session start, since anyone who can see it is immediately allowed in.

### PR DESCRIPTION
**Pull request by jducoeur**
This adds two new fields to program items with Zoom.  `doorsOpen` is the timestamp to start showing the entry links; `doorsClose` is when to stop.  (Note that doorsOpen is intentionally five minutes before the actual panel start time.)

The intent is to make it easier for the front end to show the links at the right times -- hopefully it's helpful.

For some reason, this PR is stupidly over-large: we seem to be having rebase problems.  Sorry about that.

Fixes #255

### CODE REVIEW

**Comment by clahey** [comment]
I did git merge main and it got a lot smaller and more readable.

**Comment by clahey** [comment]
Seems like a reasonable API to me. The timeStamps are the same format as timeStamp it looks like. That seems good.

**Comment by clahey** [comment]
What's the difference between doorsOpen and doorsClose and zoomStart and zoomEnd?

File: arisia-remote/conf/routes [hunk 1]
```diff
@@ -53,6 +53,30 @@ GET     /api/settings                arisia.controllers.SettingsController.getSe
...
+#  responses:
+#    200:
+#      description: OK
+#      content:
+#        text/plain:
+#          schema:
+#            type: string
+###
+GET     /api/metadata/dealers        arisia.controllers.FileController.getDealersMetadata()
+
```
Comment (Gailbear) [inline arisia-remote/conf/routes:79]: will also need events and gaming

**Review by Gailbear — APPROVED** [review]

**Pull request merged**